specification/configuration.md

```diff
@@ -163,6 +163,35 @@ Other requirements:
 - RUM library MUST use the Zipkin v2 JSON span exporter by default
 - RUM library MUST limit the number of sent spans to 100 in a 30 second window per `component` attribute value
 
+#### Serverless 
+
+Current serverless offering is composed of separate metrics (SFx-based) and traces (OTel-based) solutions. 
+It is also assumed that all the components send data directly to Splunk Observability Cloud (direct ingest). Therefore a set of specific configuration properties needs to be defined for the best user experience.
+   
+All serverless solution (wrappers, extensions and other vendor-specific ones) MUST honour following environment variables: 
+
+| Name                                   | Default value                                                              | Description                                                   |
+| -------------------------------------- | -------------------                                                        | ------------------------------------------------------------- |
+| `SPLUNK_ACCESS_TOKEN`                  | None                                                                       | Access token added to exported data.                          |
+| `SPLUNK_REALM`                         | `us0`                                                                      | Realm configured for the exporter endpoint. [1]               |
```

Comment:
I am surprised this is not part of https://github.com/signalfx/gdi-specification/blob/main/specification/configuration.md#instrumentation-libraries... isn't it required to support direct ingest?

Reply:
I guess that for instrumentation libraries URL is enough as it already contains the realm (URLs are realm-specific).